### PR TITLE
Clean storybook component

### PR DESCRIPTION
**Pull request by SaraVieira**
* Removes the need to use two storybook components and consolidates that
* Changes style of it (old vs new)
CleanShot 2021-04-23 at 20 32 08
CleanShot 2021-04-23 at 20 32 04
* Removes inline styling replacing it with CSS prop in styled components
* changes `result` to `results`
* Remove server side styled-components since it uses babel for building now

### CODE REVIEW

File: components/mdx/GridUsedBy.tsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
     <a
-      style={{
-        backgroundImage: ` url(${props.url})`,
-        backgroundSize: props.size || 'contain',
-        backgroundRepeat: 'no-repeat',
-        backgroundPosition: ' center center',
-      }}
+      {...rest}
+      css={`
+        background-image: ' url(${url})';
```
Comment (CodyJasonBennett) [inline components/mdx/GridUsedBy.tsx:6]: Is this valid CSS? I think we'll have to remove the single quotes here.

File: components/mdx/Storybook.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  <LazyLoad height={400} once>
+    <h3 className="text-xl mb-4 tracking-tight mt-6 heading">Example</h3>
+    <div className="mb-2">
+      <a
+        className="flex mt-3 items-center opacity-60 text-xs text-gray-700"
+        target="_blank"
+        rel="nofollow noopener noreferrer"
+        href={`https://${lib}.vercel.app/?path=/story/${id}`}
+      >
+        <StorybookIcon className="mr-1" /> Open in Storybook
```
Comment (CodyJasonBennett) [inline components/mdx/Storybook.tsx:14]: Let's right align interactions to the top of the story preview or the title to keep the experience consistent. I also might suggest coloring the Storybook logo if put beside the title to better communicate the hand-off.

image

**Review by CodyJasonBennett — CHANGES_REQUESTED** [review]

File: components/mdx/GridUsedBy.tsx (hunk 1 above)
Comment (SaraVieira) [reply components/mdx/GridUsedBy.tsx]: It did not, fixed

File: components/mdx/Storybook.tsx (hunk 2 above)
Comment (SaraVieira) [reply components/mdx/Storybook.tsx]: Added

File: components/mdx/Demo.tsx [hunk 3]
```diff
@@ -111,8 +111,12 @@ font-weight: bold;
         height: 400px;
       `}
     >
-      <div className="mb-2">
-        {title ? <h3 className="uppercase text-sm tracking-large text-gray-700">{title}</h3> : null}
+      <div className="flex justify-between items-center mb-2">
```
Comment (CodyJasonBennett) [inline components/mdx/Demo.tsx:114]: We don't have to return null here.

Comment (SaraVieira) [reply components/mdx/Demo.tsx:114]: Fixed in the tite and description

**Pull request merged**